stratos/chain.json

```diff
@@ -0,0 +1,108 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "stratos",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.thestratos.org",
+  "pretty_name": "Stratos",
+  "chain_id": "stratos-1",
+  "bech32_prefix": "st",
```

Comment:
Please register this prefix https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Created PR
https://github.com/satoshilabs/slips/pull/1619